doc/book/cookbook/using-zend-form-view-helpers.md

```diff
@@ -224,13 +224,17 @@ return [
         /* ... */
     ],
     'middleware_pipeline' => [
-        'pre_routing' => [
-            ['middleware' => Your\Application\FormHelpersMiddleware::class],
-            /* ... */
-        ],
-        'post_routing' => [
-            /* ... */
+        ['middleware' => Your\Application\FormHelpersMiddleware::class, 'priority' => 1000],
```

Comment:
I'm talking about merging config for `ViewHelperManager` instead of using middlewares to update it.